### DISCUSSION TITLE
No Details Tab???

### DISCUSSION THREAD

**Discussion by chriggsiii** [aclap-dev/video-downloadhelper#622] [Help and Q&A] [answered]
### Your question

In the new update for Chrome, I don't see where the old Details tab is located.  I found that very useful because it often had individual URLs for both the video and the audio.  Now all I can find is the video URL.  Where's the audio URL, please, in those cases where the audio and video are split into two feeds???  Thanks!!!

### Addon & CoApp version and details

Version 9.0.1.18 (stable), target google, locale en-US.

**Comment by paulrouget** [comment, marked as answer]
On the right of the Download button, there's a 3 dots button. Click on it. You'll see the Details section here.

**Reply by chriggsiii** [reply]
Got it; thanks!!
